### ISSUE TITLE
Sliders limited to 1024.

### ISSUE THREAD

**Issue by SirTrippsalot** [Mikubill/sd-webui-controlnet#48]
I attempted to directly resolve this myself by modifying the slider values in the py from 1024 to 4096, however it is still limited to 1024.
Almost every image I work with has a side well beyond 1024, normally more than 3x that. However even when the max for the sliders is raised to 4096, the sliders themselves are locked at 1024 max values.

**Comment by Echolink50** [comment]
> I attempted to directly resolve this myself by modifying the slider values in the py from 1024 to 4096, however it is still limited to 1024. Almost every image I work with has a side well beyond 1024, normally more than 3x that. However even when the max for the sliders is raised to 4096, the sliders themselves are locked at 1024 max values.

With the "scale to fit" selected my images are coming out whatever size I set in the standard portion of the webui. Also, if you are gonna try those super high resolutions you are gonna need a lot of VRam

**Comment by Lissanro** [comment]
I modified sliders range to 64-2048 (to be the same as in the main img2img UI) because at first I thought they can affect rendered details, but this turned out not to be the case.

When rendering 1536x1536 image on my 8GB video card (width and height set in the main img2img interface), I tried ControlNet size 64x64 and 1536x1536, and in both cases I got exactly the same output image.

Then I realized these sliders only affect input image size when I press "Open drawing canvas" to create new canvas. If I am using uploaded image, they have zero effect. They also do not affect existing canvas.

I think the reason for this limited range is because the canvas is very small window and greater resolution will not improve the result. It will be upscaled if the resolution is higher in the main UI. If there was an option to make the canvas bigger on screen, or draw in full screen, then there could be a reason to increase the range. Otherwise, I think it is good as is for small sketches.
